x-pack/qa/full-cluster-restart/src/test/java/org/elasticsearch/xpack/restart/FullClusterRestartIT.java

```diff
@@ -437,6 +442,21 @@ public void testSqlFailsOnIndexWithTwoTypes() throws IOException {
             "[testsqlfailsonindexwithtwotypes] contains more than one type [type1, type2] so it is incompatible with sql"));
     }
 
+    public void testSlmStats() throws IOException {
+        //need to stop and start ILM to ensure that SLM metadata is written to cluster state
+        if (isRunningAgainstOldCluster()) {
+            client().performRequest(new Request("POST", "_ilm/stop"));
+            client().performRequest(new Request("POST", "_ilm/start"));
```

Comment:
Because LINK, we should PUT a policy here rather than starting/stopping.

Here's an example repository/policy you can PUT that will execute every February 31 (i.e. never):
```
PUT /_snapshot/test-repo
{
  "type": "fs",
  "settings": {
    "location": "test-repo"
  }
}

PUT /_slm/policy/test-policy
{
  "schedule": "* * * 31 FEB ? *", 
  "name": "<test-snap-{now/d}>", 
  "repository": "test-repo", 
  "config": { 
    "indices": ["*"] 
  }
}
```